app/helpers/record_helper.rb

```diff
@@ -59,6 +59,19 @@ def record_media_items(record, options = {})
     Document::RecordPresenter.new(record, controller).media_web_resources(options).map(&:media_item)
   end
 
+  def record_annotations
```

Comment:
Could we add an argument to this method to accept the annotations object instead of presuming it will be in `@annotations`, to make it more reusable?